kie-remote/kie-remote-client/src/main/java/org/kie/services/client/api/command/TaskServiceClientCommandObject.java

```diff
@@ -682,6 +683,13 @@ public Task getTaskById( long taskId ) {
     }
 
     @Override
+    public List<TaskSummary> getTasksByGroup(List<String> groupIds) {
+        GetTasksByGroupCommand cmd = new GetTasksByGroupCommand();
+        cmd.getGroupIds().addAll(groupIds);
+        return executeCommand(cmd);
```

Comment:
I am not sure about this, but will ask just to double-check: do we need to handle the new command class in e.g. https://github.com/droolsjbpm/droolsjbpm-integration/blob/6.4.x/kie-remote/kie-remote-services/src/main/java/org/kie/remote/services/jaxb/JaxbCommandsResponse.java ? Similarly as the `GetTaskByWorkItemIdCommand` which was fixed there recently?